doc/basic-first-aid.rst

```diff
@@ -0,0 +1,150 @@
+      If you particularly care about any of those, you should make a copy of the :file:`.spyder-py3` folder in your user home directory, and restore it afterwards if this doesn't solve the problem.
+
+#. **Try installing Spyder into a new Conda environment** (recommended) or ``virtualenv``/``venv``, and see if the issue reoccurs.
+
+In your system terminal (or Anaconda Prompt on Windows), run the following commands to create an a fresh, clean environment and start Spyder in it:
+
+.. code-block:: bash
+
+   conda create -n spyder-env spyder -y
+   conda activate spyder-env
+   spyder
+
+   If this fixes the issue, the problem was likely due to another package installed on your system, particularly if done with pip, which can cause many problems and should be avoided if at all possible.
+
+#. **Watch our video** on solving and avoiding problems with pip, Conda and Conda-Forge, and follow its instructions.
+
+   .. youtube:: Ul79ihg41Rs
+      :height: 360
+      :width: 640
+      :align: left
+
+|
+
+
+
+.. _troubleshooting-reinstalling-spyder-ref:
+
+===================
+Reinstalling Spyder
+===================
+
+If none of the previous steps solve your issue, you should do a full uninstall of Spyder by whatever means you originally installed it.
+
+For Anaconda, follow all the steps under Option B in the `Anaconda uninstall guide`_, delete the Anaconda directory wherever it was originally installed, and (on Windows) remove the :file:`%appdata%/python` directory if it exists.
+
+.. image:: images/basic-first-aid/basic-first-aid-app-data.gif
+   :alt: Deleting appdata/python directory
+
+Then, do a clean install of the latest version of the `Anaconda distribution`_ which is how we recommend you install Spyder and keep it up to date.
+
+.. important::
+
+   While you are welcome to get Spyder working on your own by one of the many other means we offer, we are only able to provide individual support for install-related issues for users of the Anaconda distribution.
+   In particular, pip installation, while doable, is only really for experts, as there are many pitfalls involved and different issues specific to your setup, which is why we recommend using Conda whenever possible.
+   For further information, please visit our :doc:`installation`.
+
+.. _Anaconda uninstall guide: https://docs.anaconda.com/anaconda/install/uninstall/
+.. _Anaconda distribution: https://www.anaconda.com/products/individual
+
+
+
+==================
+Isolating problems
+==================
+
+If you get an error while running a specific line, block, or script/program, it may not be an issue with Spyder, but rather something lower down in the packages it depends on.
+Try running it in the following in order if and until it starts working as you expect.
+If you manage to isolate the bug, report it to the last one it *doesn't* work in.
+
+#. **Spyder** itself, of course!
+    Make sure you can reproduce the error after closing and reopening it, if possible.
```

Comment:
```suggestion
#. **Spyder** itself, of course! Make sure you can reproduce the error after closing and reopening it, if possible.
```

An odd indentation was introduced in the text due to the placement of `Make sure you can ...` (you can check it in Netlify).

Reply:
```suggestion
#. **Spyder** itself, of course!
   Make sure you can reproduce the error after closing and reopening it, if possible.
```

I was off by one space, sorry, my mistake. Here is the corrected version, retaining the one sentence per line required by the style guide.

Reply:
Agreed. I was not sure of that, so I went for the easiest solution.